docs/introduction/get-started/p2p-private.md

```diff
@@ -59,12 +59,12 @@ In this section, we show you how to make off-chain transactions and send funds t
     ```
 
     !!! tip
-        It's possible to use a short version of the note id: 7 characters after the `0x` is sufficient.
+        It's possible to use a short version of the note id: 7 characters after the `0x` is sufficient, e.g. `0x6ae613a`
 
-    You should now see both accounts containing faucet assets with half the amount transferred from `Account A` to `Account B`.
+    You should now see both accounts containing faucet assets with amounts transferred from `Account A` to `Account B`.
 
     !!! tip
-        Remember. The original amount was 100 POL.
+        Remember. The original amount was `333` tokens.
```

Comment:
i might remove this because the figures keep changing ...

Reply:
You are right, for now its correct. I have it on the radar